lib/cosmos/gui/utilities/script_module_gui.rb

```diff
@@ -134,7 +134,7 @@ def prompt_dialog_box(title, message)
       result = nil
       _get_main_thread_gui do |window|
         msg = Qt::MessageBox.new(window)
-        msg.setIcon(Qt::MessageBox::Warning)
+        msg.setIcon(Qt::MessageBox::Question)
```

Comment:
Yeah but isn't this going to affect everyone using Qt::MessageBox? Is there some change that needs to happen in qtbindings?

Reply:
It's just the icon changing from a triangle to a question mark, which is fine for COSMOS.  Question mark is probably better for the generic prompt_dialog_box anyways.   The root cause is in qtbindings mapping of constants not handling the new ::Warning class for whatever reason.